README.md

```diff
@@ -420,13 +420,72 @@ leftwm-theme apply NAME_OF_THEME_YOU_LIKE
 
 ## Without [LeftWM-Theme](https://github.com/leftwm/leftwm-theme)
 
+To set up your own custom theme, you will need to create a directory containing the theme files and then symlink it to `$XDG_CONFIG_HOME/leftwm/themes/current`.
+
+A theme directory contains at least 1 file:
+
+- `up`: a script which is loaded with the theme.
```

Comment:
Could you make a note that the scripts need to be marked as executable (`chmod u+x`) for them to work? I feel like new users forget that sometimes.